homeassistant/core.py

```diff
@@ -749,16 +749,17 @@ class Service(object):
     """Representation of a callable service."""
 
     __slots__ = ['func', 'description', 'fields', 'schema',
-                 'is_callback', 'is_coroutinefunction']
+                 'is_callback', 'is_coroutinefunction', 'state_to_set']
```

Comment:
We should only focus on state.

I'm not sure if `state_to_set` is appropriate. The parameter name should describe something about the service. So let's stick to `state`.